packages/web-components/fast-components/src/slider/slider.ts

```diff
@@ -206,8 +185,27 @@ export class Slider extends FormAssociated<HTMLInputElement>
         }
     };
 
+    private setThumbPositionForOrientation = (direction: Direction): void => {
```

Comment:
I appreciate this being factored out. Thanks!

Reply:
I did that because I had to for direction to work right  but you are welcome sir! :)